src/dss/main.py

```diff
@@ -25,7 +25,7 @@ def main():
 @main.command(name="initialize")
 @click.option(
     "--kubeconfig",
-    help=f"Path to a Kubernetes config file. Defaults to the value of the KUBECONFIG environment variable, else to '{KUBECONFIG_DEFAULT}'.",  # noqa E501
+    help=f'Content of a Kubernetes config file defining the cluster to use.  The kubeconfig will be saved to {KUBECONFIG_DEFAULT} and overwrite any previously kubeconfig stored there.  Future `dss` commands will reuse this kubeconfig by default.  Ensure you properly quote this command, for example `dss initialize --kubeconfig="$(microk8s kubeconfig)"`.',  # noqa E501
```

Comment:
Lets move the example under the whole initialize command under example section similarly as we have for `dss create`

Reply:
also want to note that correct microk8s command is `config` so `dss initialize --kubeconfig="$(microk8s config)"`

Reply:
How about now?